index.js

```diff
@@ -16,18 +16,17 @@ function relative(a, b) {
   return relativePath.charAt(0) !== '.' ? './' + relativePath : relativePath;
 }
 
-function AssetRewrite(inputTree, options) {
+function AssetRewrite(inputNode, options) {
   if (!(this instanceof AssetRewrite)) {
-    return new AssetRewrite(inputTree, options);
+    return new AssetRewrite(inputNode, options);
   }
 
-  Filter.call(this, inputTree, options);
-
   options = options || {};
+  options.extensions = options.replaceExtensions || ['html', 'css'];
+
+  Filter.call(this, inputNode, options);
 
-  this.inputTree = inputTree;
   this.assetMap = options.assetMap || {};
-  this.extensions = options.replaceExtensions || ['html', 'css'];
   this.prepend = options.prepend || '';
   this.description = options.description;
```

Comment:
Don't think so. Is there a general best practice for this with broccoli-plugins?